### ISSUE TITLE
[1.7.0.1] update React tutorial to include main.html import

### ISSUE THREAD

**Issue by srobertson421** [meteor/meteor#9939]
When starting a new meteor project from scratch, following the [react tutorial](https://www.meteor.com/tutorials/react/) yields the following error:

```
modules.js?hash=9473d030a47fbb78caa4260ae30bf578aa8b736d:2685 Uncaught Error: Target container is not a DOM element.
    at invariant (modules.js?hash=9473d030a47fbb78caa4260ae30bf578aa8b736d:2685)
    at legacyRenderSubtreeIntoContainer (modules.js?hash=9473d030a47fbb78caa4260ae30bf578aa8b736d:20820)
    at render (modules.js?hash=9473d030a47fbb78caa4260ae30bf578aa8b736d:20899)
    at Meteor.startup (main.js:8)
    at maybeReady (meteor.js?hash=c8108d734cc548d91539b054e14b613d0c4512cc:889)
    at HTMLDocument.loadingCompleted (meteor.js?hash=c8108d734cc548d91539b054e14b613d0c4512cc:901)
```

Only after adding the line: `import './main.html'` will the React application render. The react tutorial should be updated to include this.

**Comment by Geeroz** [comment]
I have to add
`import './main.html'`
to get React to render under METEOR@1.7.0.1
Have been trying to fix this for hours. Thanks @srobertson421

**Comment by aelohin** [comment]
Adding the import './main.html' works for me too.

Also, I tried what says in the documentation [https://guide.meteor.com/react.html](https://guide.meteor.com/react.html) and after adding the package static-html work as always without the need of the import of the main.html.

meteor add static-html

**Comment by srobertson421** [comment]
@aelohin Ah I see, I hadn't checked the Meteor guide, just the tutorial. Thanks for the reference!

**Comment by luchux** [comment]
Hey guys, I checked https://www.meteor.com/tutorials/react/components
Step by step, and it has the issue. 

The documentation on the tutorial should change to avoid Meteor starters to face this problem.

**Comment by antoniwan** [comment]
I just ran against this issue when attempting the simple-todos tutorial.

**Comment by jazeee** [comment]
Ditto - seems like the tutorials should be updated to deal with the static `div` topic...

**Comment by karlitos** [comment]
I noticed different effect. After removing _blaze-html-templates_ and adding _static-html_ meteor packages you have remove the `import './main.html` otherwise meteor will complain it cannot find module main.html. Hope that helps somebody.

**Comment by mcbriandavids** [comment]
Nice comment but I believe the best way to go about such error is my removing the default blaze-html-template the follows the creation by simply using your command line(command prompt as the case may) to "meteor remove blaze-html-template" and then "meteor add static-html". I believe this approach will minimize the in the code better.

**Comment by mcbriandavids** [comment]
A few persons have complained about changing or updating the tutorial, however, i was of same opinion too but the fact remains that if things are made extremely easy, we don't discover new things. The challenges I had in my quest for solution broaden my understanding of the architecture of meteor better.

**Comment by jazeee** [comment]
I think this issue is really problematic.

The Meteor + React tutorial should work "out of the box" for someone who is just starting to play with Meteor.
I just went through first part of the tutorial, and even though I have used Meteor, on and off, for a few years, I found it bizarre that the tutorial doesn't work.

Simply said, It makes Meteor look bad.

The tutorial is quite good otherwise, and guides developers quite well, in every aspect except for this one issue.

To be clear, the solution is to run the following:
```
meteor remove blaze-html-templates
meteor add static-html
```
